tools/skel-pack/package.js

```diff
@@ -1,8 +1,13 @@
 Package.describe({
   name: '~name~',
-  summary: ' /* Fill me in! */ ',
-  version: '1.0.0',
-  git: ' /* Fill me in! */ '
+  // Brief, one-line summary of the package.
+  summary: '',
+  version: '0.0.1',
+  // By default, Meteor will default to using README.md for documentation. Use
```

Comment:
I think you have a good point, and I think I have a good point, and they are in conflict.  Maybe a third opinion here would be helpful.